tests/plugins/widget/manual/outline.html

```diff
@@ -26,6 +26,11 @@ <h2>Inline</h2>
 
 
 <script>
+	// Inline editor with widget crashes on IE8 #1641 & #1821
+	if ( !( CKEDITOR.env.ie && CKEDITOR.env.version > 9 ) ) {
```

Comment:
Doing such conditions is pretty unreadable. You could improve readability by reversing it.
`CKEDITOR.env.ie && CKEDITOR.env.version < 9`